resources/telemetry/charts/fluent-bit/values.yaml

```diff
@@ -374,12 +375,12 @@ config:
         Refresh_Interval 10
         DB /data/flb_kube.db
         storage.type  filesystem
-
+ 
     [INPUT]
         Name tail
         Path /null.log
         Tag null.*
-        Alias dummy
+        Alias tail-dummy
```

Comment:
Shouldn't it be called "null-dummy" as well as it is part of the "null" pipeline somehow